src/app/(app)/home/entries/[slug]/page.jsx

```diff
@@ -0,0 +1,49 @@
+import { getEntry } from '@/app/actions/entries.js';
+import { getEntries } from '@/app/actions/entries.js';
+import styles from './viewer.module.scss';
+
+export async function generateStaticParams() {
+  // TODO: replace user id
+  const entries = await getEntries(1);
+
+  return entries.map((entry) => ({
+    slug: entry.slug,
+  }));
+}
+
+export default async function Page({ params }) {
+  const entryResponse = await getEntry(params.slug);
+  const entry = entryResponse[0];
+
+  return (
+    <section className={styles.viewerContainer}>
```

Comment:
The use of section elements may be overused here. Personally I find the use of section to be for easy style targeting. (e.g. `.pageWrapper { section{...}}`)

Reply:
I used section and div pretty arbitrarily here, thanks for pointing this part out. I'll probably try to use the section tag in a more semantic way, and divs for when the element is only for styling purposes